lib/savings/save_money_with_bucket/save_money_with_bucket.dart

```diff
@@ -0,0 +1,223 @@
+      backgroundColor: Theme.of(context).cardColor,
+      body: LayoutBuilder(
+        builder: (context, BoxConstraints box) {
+          return Stack(
+            children: [
+              SingleChildScrollView(
+                child: Column(
+                  children: [
+                    Row(
+                      children: [
+                        IconButton(
+                          icon: SvgPicture.asset(
+                              'assets/FpbIcons/left_arrow.svg'),
+                          onPressed: () {},
+                        ),
+                        Expanded(
+                          child: Center(
+                            child: Text(
+                              l10n.saveMoneyWithBucketScreenTitle,
+                              style: style.titleMedium?.copyWith(
+                                fontWeight: FontWeight.w600,
+                              ),
+                            ),
+                          ),
+                        ),
+                      ],
+                    ),
+                    SizedBox(
+                      height: box.maxHeight * 0.05,
+                    ),
+                    Row(
+                      children: [
+                        IconButton(
+                          icon: SvgPicture.asset(
+                              'assets/FpbIcons/dollar_sign.svg'),
+                          onPressed: () {},
+                        ),
+                        Expanded(
+                          child: Center(
+                            child: TextFormField(
+                              controller: amountController,
+                              readOnly: true,
+                              style: style.titleLarge?.copyWith(
+                                fontWeight: FontWeight.w600,
+                                fontSize: 40,
+                              ),
+                              keyboardType: TextInputType.number,
+                              textAlign: TextAlign.center,
+                              decoration: InputDecoration(
+                                border: InputBorder.none,
+                                focusedBorder: InputBorder.none,
+                                enabledBorder: InputBorder.none,
+                                errorBorder: InputBorder.none,
+                                disabledBorder: InputBorder.none,
+                                hintText: "00.0",
+                                hintStyle: style.titleLarge?.copyWith(
+                                  fontWeight: FontWeight.w600,
+                                  fontSize: 40,
+                                ),
+                              ),
+                            ),
+                          ),
+                        ),
+                      ],
+                    ),
+                    SizedBox(
+                      height: box.maxHeight * 0.03,
+                    ),
+                    Card(
```

Comment:
You should use Styled `Container` with boxShadow instead of `Card` to reflect Rudy's design. You can have the shadow params on Figma to apply on the `Container` Widget.
😅 I don't know if appearance(with respect to design on Figma) is part of the acceptance Criteria though?
What do you think @lkmandy , @fonkamloic

Reply:
After attempting to use the `styled container `approach, it seems the container's background needs to be different from the scaffold's background for the elevation effect to be visible but in this case, both are white.

I'm able to achieve this with the elevation property in the card widget.
From this StackOverflow guide https://bit.ly/3JbjzAX it seems to be a decent approach

Reply:
Please can you share a screenshot? 
also with the styled Container you can follow the styles property (color, drop-shadow, etc) of the component from the Figma design.
because I think with the elevation property you won't get the desired effect from the initial design.